multiqc/utils/search_patterns.yaml

```diff
@@ -363,6 +361,9 @@ longranger/invocation:
   max_filesize: 2048
 macs2:
   fn: "*_peaks.xls"
+sambamba/markdup:
+  fn: "markdup_*"
+  contents: "finding positions of the duplicate reads in the file"
```

Comment:
If this is always at the top of the file, then please set `num_lines: 5` or something to stop MultiQC crawling through huge logs.